backend/typescript/middlewares/validators/campValidators.ts

```diff
@@ -379,6 +386,56 @@ export const deleteCampSessionsDtoValidator = async (
   return next();
 };
 
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable-next-line import/prefer-default-export */
+export const updateCampSessionsDtoValidator = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  const campSessions = req.body.updatedCampSessions;
```

Comment:
Is it possible to enforce a CampSession type here?

Reply:
Hmm we haven't brought types into this file before, didn't really wanna mess with that pattern?

Reply:
fair enough